src/main/java/com/example/pet_shelter/controller/PersonRESTController.java

```diff
@@ -16,36 +17,41 @@ public class PersonRESTController {
     private PersonService personService;
 
     @GetMapping("/persons")
+    @Cacheable(value = "ShowingAllPersons")
     public List<Person> showAllPersons() {
         List<Person> allPerosns = personService.getAllPersons();
         return allPerosns;
     }
 
     @GetMapping("/persons/{id}")
+    @Cacheable (value = "personID")
     public Person getPerson(@PathVariable int id) {
         Person person = personService.getPerson(id);
         return person;
     }
 
 
     @PostMapping("/persons")
+    @Cacheable (value = "personNameAdded")
```

Comment:
not a good idea to add caching for post, in this case it probably will not update person, hust cache it it redis